### ISSUE TITLE
PPC pretrained weights

### ISSUE THREAD

**Issue by trungpham2606** [mrlooi/maskrcnn-benchmark#20]
## ❓ Questions and Help
@mrlooi 
Can you public your pretrained weights on PPC dataset ?

**Comment by mrlooi** [comment]
Hey unfortunately I don't have the pretrained weights with me. In any case the MSCOCO pretrained weights should be more better - it's more general

**Comment by trungpham2606** [comment]
@mrlooi 
In addition,
You have not published your paper yet, have you ?

**Comment by mrlooi** [comment]
Nope, it takes work, and doubt it's going to make much of an impact anyway. I'm not a full time researcher, so publications and citations are not as important to me xD

**Comment by trungpham2606** [comment]
@mrlooi 
I often meet this error after some training iterations:
![image](https://user-images.githubusercontent.com/35657426/75305461-c6a14900-5878-11ea-83d6-7d7238b2f02d.png)
Can you tell me how to solve this issue ?

**Comment by mrlooi** [comment]
It would seem that the input batch size is empty (**0** x 1 x ...). A good way to check is backtrack where the input became empty. I haven't seen this issue before, but if you do find out the cause, please let me know

**Comment by trungpham2606** [comment]
@mrlooi 
Yeah sure, Iam training with my own dataset to see if this error still occurs.

**Comment by trungpham2606** [comment]
@mrlooi 
I think the error comes from the model not about the dataset. 
For PPC dataset, i had printed out the images when the errors come out.
![image](https://user-images.githubusercontent.com/35657426/75312672-4507e600-588d-11ea-9cd0-c8ce6e95fd41.png)
![image](https://user-images.githubusercontent.com/35657426/75312688-4cc78a80-588d-11ea-8584-4d9a488ec9da.png)

The error comes with different images each times.
The objects inside the images I guess they're not too small.
Do you have any ideas about this ?

**Comment by mrlooi** [comment]
Hey good question. I think it could be that the ground truth rotated bboxes for those objects did not have good IoU fit with any of the rotated anchors

**Comment by trungpham2606** [comment]
@mrlooi 
I had changed some threshold for being able to train PPC dataset.
Thank you for your prompt support!

**Comment by trungpham2606** [comment]
@mrlooi 
Excuse me but can you please give me some sources of reference you had used in your work ?
Does RRPN you used come from this paper: Arbitrary-Oriented Scene Text Detection via
Rotation Proposals ?(https://arxiv.org/pdf/1703.01086.pdf)

**Comment by mrlooi** [comment]
Yes RRPN and MaskRCNN are the main ones

**Comment by elepherai** [comment]
> @mrlooi
> I had changed some threshold for being able to train PPC dataset.
> Thank you for your prompt support!

@trungpham2606 Hi, could you tell me what thresholds did you change for training PPC dataset? Thanks~
